packages/web/src/common/store/pages/signon/sagas.ts

```diff
@@ -246,31 +270,35 @@ function* validateHandle(action) {
       ) ?? DEFAULT_HANDLE_VERIFICATION_TIMEOUT_MILLIS
 
     if (ENVIRONMENT === 'production') {
-      const results = yield all([
-        remoteConfigInstance.getFeatureEnabled(
-          FeatureFlags.VERIFY_HANDLE_WITH_TWITTER
-        )
-          ? race({
-              data: call(audiusBackendInstance.twitterHandle, handle),
-              timeout: delay(handleCheckTimeout)
-            })
-          : null,
-        remoteConfigInstance.getFeatureEnabled(
-          FeatureFlags.VERIFY_HANDLE_WITH_INSTAGRAM
-        )
-          ? race({
-              data: call(audiusBackendInstance.instagramHandle, handle),
-              timeout: delay(handleCheckTimeout)
-            })
-          : null,
-        remoteConfigInstance.getFeatureEnabled(
-          FeatureFlags.VERIFY_HANDLE_WITH_TIKTOK
-        )
-          ? race({
-              data: call(audiusBackendInstance.tiktokHandle, handle),
-              timeout: delay(handleCheckTimeout)
-            })
-          : null
+      const verifyTwitter = remoteConfigInstance.getFeatureEnabled(
+        FeatureFlags.VERIFY_HANDLE_WITH_TWITTER
+      )
+      const verifyInstagram = remoteConfigInstance.getFeatureEnabled(
+        FeatureFlags.VERIFY_HANDLE_WITH_INSTAGRAM
+      )
+      const verifyTikTok = remoteConfigInstance.getFeatureEnabled(
+        FeatureFlags.VERIFY_HANDLE_WITH_TIKTOK
+      )
+
+      const results = yield* all([
+        race({
+          data: verifyTwitter
+            ? call(audiusBackendInstance.twitterHandle, handle)
+            : null,
+          timeout: delay(handleCheckTimeout)
+        }),
+        race({
+          data: verifyInstagram
+            ? call(audiusBackendInstance.instagramHandle, handle)
+            : null,
+          timeout: delay(handleCheckTimeout)
+        }),
+        race({
+          data: verifyTikTok
+            ? call(audiusBackendInstance.tiktokHandle, handle)
+            : null,
+          timeout: delay(handleCheckTimeout)
+        })
```

Comment:
Reorganized this code a bit to help typescript be able to parse the types better. Small functional change to always race and return the object, but should not have an impact on the flow

Reply:
🥇